services/replicator/metadataReconciler.go

```diff
@@ -333,7 +333,7 @@ func (r *metadataReconciler) reconcileCg(localCgs []*shared.ConsumerGroupDescrip
 			replicatorReconcileCgFoundMissingCount = replicatorReconcileCgFoundMissingCount + 1
 
 			// If the missing ConsumerGroup is in deleted status, we don't need to create the ConsumerGroup locally
-			if remoteCg.GetStatus() == shared.ConsumerGroupStatus_DELETED {
+			if remoteCg.GetStatus() == shared.ConsumerGroupStatus_DELETING {
```

Comment:
I think we should check for both conditions(deleted/deleting). If remote is in deleted, we also don't need to create it locally.

Reply:
CGs in "DELETED" state are *not* returned by the ListConsumerGroups API .. and are technically 'invisible'. So that check (on a DELETED CG) would be pointless.